### ISSUE TITLE
I want  to Publish my app To AppSource but i get this Error Message: Cannot bind argument to parameter 'authContext'

### ISSUE THREAD

**Issue by fofanasbsi** [microsoft/AL-Go#829]
Hello Freddy,
I've been trying to migrate my app in AppSource in the CI/CD under github actions, but I'm getting this error.
I don't know what mistake I'm making.
```
Error: Unexpected error when running action. Error Message: Cannot bind argument to parameter 'authContext' because it is null., StackTrace: at <ScriptBlock>, D:\a\_actions\microsoft\AL-Go-Actions\v4.0\Deliver\Deliver.ps1: line 467 <- at <ScriptBlock>, D:\a\_temp\d055374f-e5ff-4f35-bea8-30781453e01c.ps1: line 4 <- at <ScriptBlock>, <No file>: line 1 
[87](https://github.com/fofanasbsi/AppSource/actions/runs/6903051034/job/18780979659#step:5:90)Error: Process completed with exit code 1.

Run microsoft/AL-Go-Actions/Deliver@v4.0 
[46](https://github.com/fofanasbsi/AppSource/actions/runs/6903051034/job/18780979659#step:5:46) with: 
[47](https://github.com/fofanasbsi/AppSource/actions/runs/6903051034/job/18780979659#step:5:47) shell: powershell 
[48](https://github.com/fofanasbsi/AppSource/actions/runs/6903051034/job/18780979659#step:5:48) parentTelemetryScopeJson: 7b22506172616d6574657273223a7b7d2c22456d6974746564223a66616c73652c22436f7272656c6174696f6e4 
[49](https://github.com/fofanasbsi/AppSource/actions/runs/6903051034/job/18780979659#step:5:49) type: Release 
[50](https://github.com/fofanasbsi/AppSource/actions/runs/6903051034/job/18780979659#step:5:50) projects: * 
[51](https://github.com/fofanasbsi/AppSource/actions/runs/6903051034/job/18780979659#step:5:51) deliveryTarget: AppSource 
[52](https://github.com/fofanasbsi/AppSource/actions/runs/6903051034/job/18780979659#step:5:52) artifacts: current 
[53](https://github.com/fofanasbsi/AppSource/actions/runs/6903051034/job/18780979659#step:5:53) goLive: N 
[54](https://github.com/fofanasbsi/AppSource/actions/runs/6903051034/job/18780979659#step:5:54) actor: fofanasbsi 
[55](https://github.com/fofanasbsi/AppSource/actions/runs/6903051034/job/18780979659#step:5:55) token: *** 
[56](https://github.com/fofanasbsi/AppSource/actions/runs/6903051034/job/18780979659#step:5:56) atypes: Apps,Dependencies,TestApps 
[57](https://github.com/fofanasbsi/AppSource/actions/runs/6903051034/job/18780979659#step:5:57) env: 
[58](https://github.com/fofanasbsi/AppSource/actions/runs/6903051034/job/18780979659#step:5:58) ALGoOrgSettings: 
[59](https://github.com/fofanasbsi/AppSource/actions/runs/6903051034/job/18780979659#step:5:59) ALGoRepoSettings: 
[60](https://github.com/fofanasbsi/AppSource/actions/runs/6903051034/job/18780979659#step:5:60) Settings: {"keyVaultCertificateUrlSecretName":"","microsoftTelemetryConnectionString":"InstrumentationKey=84bd9223-67d4-4378-8590-9e4a46023be2;IngestionEndpoint=https://westeurope-1.in.applicationinsights.azure.com/","type":"AppSource App","sendExtendedTelemetryToMicrosoft":false,"appBuild":7,"installTestApps":[],"versioningStrategy":0,"installApps":[],"installTestLibraries":false,"runNumberOffset":0,"codeSignCertificatePasswordSecretName":"codeSignCertificatePassword","vsixFile":"","country":"us","partnerTelemetryConnectionString":"","doNotRunTests":false,"githubRunner":"windows-latest","generateDependencyArtifact":true,"excludeEnvironments":[],"keyVaultCertificatePasswordSecretName":"","rulesetFile":"","licenseFileUrlSecretName":"licenseFileUrl","enableExternalRulesets":false,"memoryLimit":"","keyVaultClientIdSecretName":"","testFolders":[],"doNotBuildTests":false,"environments":[],"companyName":"","assignPremiumPlan":false,"artifact":"","ghTokenWorkflowSecretName":"ghTokenWorkflow","enableTaskScheduler":false,"enableCodeCop":false,"repoVersion":"1.0","doNotPublishApps":false,"unusedALGoSystemFiles":[],"githubRunnerShell":"powershell","appFolders":[],"runs-on":"windows-latest","cacheKeepDays":3,"bcptTestFolders":[],"updateDependencies":false,"cacheImageName":"my","treatTestFailuresAsWarnings":false,"AppSourceProductId":"adafcdca-c504-4fb6-8b23-081d4a7c0922","additionalCountries":[],"enableUICop":false,"customCodeCops":[],"appDependencies":[],"doNotRunBcptTests":false,"appDependencyProbingPaths":[],"useCompilerFolder":false,"installTestFramework":false,"keyVaultCodesignCertificateName":"","keyVaultName":"","installOnlyReferencedApps":true,"obsoleteTagMinAllowedMajorMinor":"","applicationDependency":"18.0.0.0","applicationInsightsConnectionStringSecretName":"applicationInsightsConnectionString","shell":"powershell","failOn":"error","templateUrl":"https://github.com/microsoft/AL-Go-AppSource@main","projects":[],"projectName":".","testDependencies":[],"appRevision":0,"codeSignCertificateUrlSecretName":"codeSignCertificateUrl","configPackages":[],"useProjectDependencies":false,"appSourceCopMandatoryAffixes":["\u003caffix\u003e","BingMaps"],"buildModes":[],"templateBranch":"","installPerformanceToolkit":false,"PullRequestTrigger":"pull_request_target","doNotSignApps":false,"repoName":"AppSource","fullBuildPatterns":[],"alwaysBuildAllProjects":false,"skipUpgrade":false,"installTestRunner":false,"adminCenterApiCredentialsSecretName":"adminCenterApiCredentials"} 
[61](https://github.com/fofanasbsi/AppSource/actions/runs/6903051034/job/18780979659#step:5:61) Secrets: {"appSourceContext":""}
```

**Comment by freddydk** [comment]
If you could include the full log of the deliver action, then I might be able to see what's wrong.
My assumption is that your AppSourceContext secret is somehow wrong

**Comment by Steven-SBSI** [comment]
Hello,

Here is the full log, is it enough or do you need more details ?

Let me know.

**Comment by Steven-SBSI** [comment]
Run microsoft/AL-Go-Actions/Deliver@v4.0
  with:
    shell: powershell
    parentTelemetryScopeJson: 7b[2](https://github.com/fofanasbsi/AppSource/actions/runs/6905165127/job/18788521262#step:5:2)2506172616d657465727[3](https://github.com/fofanasbsi/AppSource/actions/runs/6905165127/job/18788521262#step:5:3)223a7b7d2c22[4](https://github.com/fofanasbsi/AppSource/actions/runs/6905165127/job/18788521262#step:5:4)[5](https://github.com/fofanasbsi/AppSource/actions/runs/6905165127/job/18788521262#step:5:5)[6](https://github.com/fofanasbsi/AppSource/actions/runs/6905165127/job/18788521262#step:5:6)d69[7](https://github.com/fofanasbsi/AppSource/actions/runs/6905165127/job/18788521262#step:5:7)4746564223a66616c73652c22436f7272656c6174696f6e4964223a2232656561306132332d313531302d346363302d613434372d663637313[8](https://github.com/fofanasbsi/AppSource/actions/runs/6905165127/job/18788521262#step:5:8)30623334303763222c22506172656e744[9](https://github.com/fofanasbsi/AppSource/actions/runs/6905165127/job/18788521262#step:5:9)64223a22222c22416c6c506172616d6574657273223a7b7d2c22546f704964223a2232656561306132332d313531302d346363302d613434372d663637313830623334303763222c224e616d65223a22414c2d476f20776f726b666c6f772072616e3a205075626c697368546f417070536f75726365222c22537461727454696d65223a225c2f446174652831373030323330383538393232295c2f222c2250726f70657274696573223a7b226576656e744964223a22444f30313033222c226263436f6e7461696e657248656c70657256657273696f6e223a22362e302e30222c22697341646d696e6973747261746f72223a2254727565222c22737461636b5472616365223a225b5c6e2020496e697454656c656d6574727953636f70652061742054656c656d6574727948656c7065722e7073313a206c696e65203231345c6e202043726561746553636f70652061742054656c656d6574727948656c7065722e70736d313a206c696e652037355c6e2020576f726b666c6f77496e697469616c697a652e70733120617420576f726b666c6f77496e697469616c697a652e7073313a206c696e652033355c6e202032626236383830322d666263362d346234352d623538612d6633663533663731636632302e7073312061742032626236383830322d666263362d346234352d623538612d6633663533663731636632302e7073313a206c696e6520345c6e20205c7530303363536372697074426c6f636b5c7530303365206174205c75303033634e6f2066696c655c75303033655c6e5d222c22414c476f56657273696f6e223a2276342e30222c2274797065223a22417070536f7572636520417070222c2274656d706c61746555726c223a2268747470733a2f2f6769746875622e636f6d2f6d6963726f736f66742f414c2d476f2d417070536f75726365406d61696e222c227265706f7369746f7279223a22666f66616e61736273692f417070536f75726365222c2272756e417474656d7074223a2231222c2272756e4e756d626572223a2239222c2272756e4964223a2236393035313635313237227d2c224576656e744964223a22444f30313033227d
    type: Release
    projects: *
    deliveryTarget: AppSource
    artifacts: latest
    goLive: N
    actor: fofanasbsi
    token: ***
    atypes: Apps,Dependencies,TestApps
  env:
    ALGoOrgSettings: 
    ALGoRepoSettings: 
    Settings: {"keyVaultCertificateUrlSecretName":"","microsoftTelemetryConnectionString":"InstrumentationKey=84bd9223-67d4-4378-8590-9e4a46023be2;IngestionEndpoint=https://westeurope-1.in.applicationinsights.azure.com/","type":"AppSource App","sendExtendedTelemetryToMicrosoft":false,"appBuild":9,"installTestApps":[],"versioningStrategy":0,"installApps":[],"installTestLibraries":false,"runNumberOffset":0,"codeSignCertificatePasswordSecretName":"codeSignCertificatePassword","country":"us","excludeEnvironments":[],"partnerTelemetryConnectionString":"","doNotRunTests":false,"runs-on":"windows-latest","configPackages":[],"generateDependencyArtifact":true,"templateUrl":"https://github.com/microsoft/AL-Go-AppSource@main","keyVaultCertificatePasswordSecretName":"","rulesetFile":"","licenseFileUrlSecretName":"licenseFileUrl","enableExternalRulesets":false,"memoryLimit":"","keyVaultClientIdSecretName":"","testFolders":[],"githubRunner":"windows-latest","authContext":{"TenantID":"dec36c5a-abfb-4e08-b021-3adae1cf7b1d","ClientID":"ed6d0115-ebe8-42fe-a0ce-92a7134ec5cb","ClientSecret":"***"},"doNotBuildTests":false,"environments":[],"companyName":"","assignPremiumPlan":false,"artifact":"","ghTokenWorkflowSecretName":"ghTokenWorkflow","enableTaskScheduler":false,"enableCodeCop":false,"repoVersion":"1.0","doNotPublishApps":false,"doNotSignApps":false,"githubRunnerShell":"powershell","cacheKeepDays":3,"bcptTestFolders":[],"updateDependencies":false,"cacheImageName":"my","treatTestFailuresAsWarnings":false,"AppSourceProductId":"***","additionalCountries":[],"enableUICop":false,"customCodeCops":[],"appDependencies":[],"doNotRunBcptTests":false,"skipUpgrade":false,"appDependencyProbingPaths":[],"useCompilerFolder":false,"installTestFramework":false,"keyVaultCodesignCertificateName":"","obsoleteTagMinAllowedMajorMinor":"","applicationDependency":"18.0.0.0","applicationInsightsConnectionStringSecretName":"applicationInsightsConnectionString","shell":"powershell","keyVaultName":"","failOn":"error","projects":[],"appSourceCopMandatoryAffixes":["\u003caffix\u003e","BingMaps"],"projectName":".","testDependencies":[],"appRevision":2,"codeSignCertificateUrlSecretName":"codeSignCertificateUrl","vsixFile":"","useProjectDependencies":false,"unusedALGoSystemFiles":[],"buildModes":[],"templateBranch":"","installPerformanceToolkit":false,"PullRequestTrigger":"pull_request_target","appFolders":[],"repoName":"AppSource","fullBuildPatterns":[],"alwaysBuildAllProjects":false,"installOnlyReferencedApps":true,"installTestRunner":false,"adminCenterApiCredentialsSecretName":"adminCenterApiCredentials"}
    Secrets: {"appSourceContext":"***"}
Run $errorActionPreference = "Stop"; $ProgressPreference = "SilentlyContinue"; Set-StrictMode -Version 2.0
  $errorActionPreference = "Stop"; $ProgressPreference = "SilentlyContinue"; Set-StrictMode -Version 2.0
  try {
    D:\a\_actions\microsoft\AL-Go-Actions\v4.0\Deliver/Deliver.ps1 -actor $ENV:_actor -token $ENV:_token -parentTelemetryScopeJson $ENV:_parentTelemetryScopeJson -projects $ENV:_projects -deliveryTarget $ENV:_deliveryTarget -artifacts $ENV:_artifacts -type $ENV:_type -atypes $ENV:_atypes -goLive ($ENV:_goLive -eq 'Y')
  }
  catch {
    Write-Host "::ERROR::Unexpected error when running action. Error Message: $($_.Exception.Message.Replace("`r",'').Replace("`n",' ')), StackTrace: $($_.ScriptStackTrace.Replace("`r",'').Replace("`n",' <- '))";
    exit 1
  }
  shell: C:\Windows\System32\WindowsPowerShell\v1.0\powershell.EXE -command ". '{0}'"
  env:
    ALGoOrgSettings: 
    ALGoRepoSettings: 
    Settings: {"keyVaultCertificateUrlSecretName":"","microsoftTelemetryConnectionString":"InstrumentationKey=84bd9223-67d4-4378-8590-9e4a46023be2;IngestionEndpoint=https://westeurope-1.in.applicationinsights.azure.com/","type":"AppSource App","sendExtendedTelemetryToMicrosoft":false,"appBuild":9,"installTestApps":[],"versioningStrategy":0,"installApps":[],"installTestLibraries":false,"runNumberOffset":0,"codeSignCertificatePasswordSecretName":"codeSignCertificatePassword","country":"us","excludeEnvironments":[],"partnerTelemetryConnectionString":"","doNotRunTests":false,"runs-on":"windows-latest","configPackages":[],"generateDependencyArtifact":true,"templateUrl":"https://github.com/microsoft/AL-Go-AppSource@main","keyVaultCertificatePasswordSecretName":"","rulesetFile":"","licenseFileUrlSecretName":"licenseFileUrl","enableExternalRulesets":false,"memoryLimit":"","keyVaultClientIdSecretName":"","testFolders":[],"githubRunner":"windows-latest","authContext":{"TenantID":"dec36c5a-abfb-4e08-b021-3adae1cf7b1d","ClientID":"ed6d0115-ebe8-42fe-a0ce-92a7134ec5cb","ClientSecret":"***"},"doNotBuildTests":false,"environments":[],"companyName":"","assignPremiumPlan":false,"artifact":"","ghTokenWorkflowSecretName":"ghTokenWorkflow","enableTaskScheduler":false,"enableCodeCop":false,"repoVersion":"1.0","doNotPublishApps":false,"doNotSignApps":false,"githubRunnerShell":"powershell","cacheKeepDays":3,"bcptTestFolders":[],"updateDependencies":false,"cacheImageName":"my","treatTestFailuresAsWarnings":false,"AppSourceProductId":"***","additionalCountries":[],"enableUICop":false,"customCodeCops":[],"appDependencies":[],"doNotRunBcptTests":false,"skipUpgrade":false,"appDependencyProbingPaths":[],"useCompilerFolder":false,"installTestFramework":false,"keyVaultCodesignCertificateName":"","obsoleteTagMinAllowedMajorMinor":"","applicationDependency":"18.0.0.0","applicationInsightsConnectionStringSecretName":"applicationInsightsConnectionString","shell":"powershell","keyVaultName":"","failOn":"error","projects":[],"appSourceCopMandatoryAffixes":["\u003caffix\u003e","BingMaps"],"projectName":".","testDependencies":[],"appRevision":2,"codeSignCertificateUrlSecretName":"codeSignCertificateUrl","vsixFile":"","useProjectDependencies":false,"unusedALGoSystemFiles":[],"buildModes":[],"templateBranch":"","installPerformanceToolkit":false,"PullRequestTrigger":"pull_request_target","appFolders":[],"repoName":"AppSource","fullBuildPatterns":[],"alwaysBuildAllProjects":false,"installOnlyReferencedApps":true,"installTestRunner":false,"adminCenterApiCredentialsSecretName":"adminCenterApiCredentials"}
    Secrets: {"appSourceContext":"***"}
    _actor: fofanasbsi
    _token: ***
    _parentTelemetryScopeJson: 7b22506172616d6574657273223a7b7d2c22456d6974746564223a66616c73652c22436f7272656c6174696f6e4964223a2232656561306132332d313531302d346363302d613434372d663637313830623334303763222c22506172656e744964223a22222c22416c6c506172616d6574657273223a7b7d2c22546f704964223a2232656561306132332d313531302d346363302d613434372d663637313830623334303763222c224e616d65223a22414c2d476f20776f726b666c6f772072616e3a205075626c697368546f417070536f75726365222c22537461727454696d65223a225c2f446174652831373030323330383538393232295c2f222c2250726f70657274696573223a7b226576656e744964223a22444f30313033222c226263436f6e7461696e657248656c70657256657273696f6e223a22362e302e30222c22697341646d696e6973747261746f72223a2254727565222c22737461636b5472616365223a225b5c6e2020496e697454656c656d6574727953636f70652061742054656c656d6574727948656c7065722e7073313a206c696e65203231345c6e202043726561746553636f70652061742054656c656d6574727948656c7065722e70736d313a206c696e652037355c6e2020576f726b666c6f77496e697469616c697a652e70733120617420576f726b666c6f77496e697469616c697a652e7073313a206c696e652033355c6e202032626236383830322d666263362d346234352d623538612d6633663533663731636632302e7073312061742032626236383830322d666263362d346234352d623538612d6633663533663731636632302e7073313a206c696e6520345c6e20205c7530303363536372697074426c6f636b5c7530303365206174205c75303033634e6f2066696c655c75303033655c6e5d222c22414c476f56657273696f6e223a2276342e30222c2274797065223a22417070536f7572636520417070222c2274656d706c61746555726c223a2268747470733a2f2f6769746875622e636f6d2f6d6963726f736f66742f414c2d476f2d417070536f75726365406d61696e222c227265706f7369746f7279223a22666f66616e61736273692f417070536f75726365222c2272756e417474656d7074223a2231222c2272756e4e756d626572223a2239222c2272756e4964223a2236393035313635313237227d2c224576656e744964223a22444f30313033227d
    _projects: *
    _deliveryTarget: AppSource
    _artifacts: latest
    _type: Release
    _atypes: Apps,Dependencies,TestApps
    _goLive: N
Downloading BcContainerHelper latest version from CDN
Import from C:\ProgramData\BcContainerHelper\6.0.0\BcContainerHelper\BcContainerHelper.ps1
BcContainerHelper version 6.0.0
BC.HelperFunctions emits usage statistics telemetry to Microsoft
Applying settings from D:\a\AppSource\AppSource\.github\AL-Go-Settings.json
Applying settings from D:\a\AppSource\AppSource\.AL-Go\settings.json
No settings found in D:\a\AppSource\AppSource\.github\Publish To AppSource.settings.json
No settings found in D:\a\AppSource\AppSource\.AL-Go\Publish To AppSource.settings.json
No settings found in D:\a\AppSource\AppSource\.AL-Go\fofanasbsi.settings.json
Artifacts latest
Projects:
.
Project 'AppSource'
Analyzing artifacts matching *-main-*-*
https://api.github.com/repos/fofanasbsi/AppSource/actions/artifacts?per_page=[10](https://github.com/fofanasbsi/AppSource/actions/runs/6905165127/job/18788521262#step:5:10)0&page=1
Matching artifacts:
- AppSource-main-Apps-1.0.19.0
Downloading artifact AppSource-main-Apps-1.0.19.0
https://api.github.com/repos/fofanasbsi/AppSource/actions/artifacts/105724[13](https://github.com/fofanasbsi/AppSource/actions/runs/6905165127/job/18788521262#step:5:13)45/zip
D:\a\AppSource\AppSource\.artifacts\AppSource-main-Apps-1.0.19.0.zip
Analyzing artifacts matching *-main-*-*
https://api.github.com/repos/fofanasbsi/AppSource/actions/artifacts?per_page=100&page=1
Matching artifacts:
- No matching artifacts found
WARNING: Could not find any Dependencies artifacts for projects *, version latest
Artifacts:
- AppSource-main-Apps-1.0.19.0
Applying settings from D:\a\AppSource\AppSource\.github\AL-Go-Settings.json
Applying settings from D:\a\AppSource\AppSource\.AL-Go\settings.json
No settings found in D:\a\AppSource\AppSource\.github\Publish To AppSource.settings.json
No settings found in D:\a\AppSource\AppSource\.AL-Go\Publish To AppSource.settings.json
No settings found in D:\a\AppSource\AppSource\.AL-Go\fofanasbsi.settings.json
Analyzing repository
  Checking type
  Checking appFolders, testFolders and bcptTestFolders
  Application Dependency 2***.0
  Updating app- and test Dependencies
  Analyzing Test App Dependencies
No performance test apps found in bcptTestFolders in .AL-Go\settings.json
Azure.Storage Module is available in version 2.1.0
Using Azure.Storage version 2.1.0

Authentication failed
AppSource MainAppFolder .\AppSourceWorkFlow
Main App File:
- MohamedFOFANA_AppSourceWorkFlow_1.0.19.0.app
Library App Files:
- None
Submitting to AppSource
AL-Go action ran: Deliver Telemetry Correlation Id: 2a9c1a20-13e2-46ee-9381-f3[14](https://github.com/fofanasbsi/AppSource/actions/runs/6905165127/job/18788521262#step:5:14)d6c5e66d
Error: Unexpected error when running action. Error Message: Cannot bind argument to parameter 'authContext' because it is null., StackTrace: at <ScriptBlock>, D:\a\_actions\microsoft\AL-Go-Actions\v4.0\Deliver\Deliver.ps1: line 467 <- at <ScriptBlock>, D:\a\_temp\1b3f568e-[26](https://github.com/fofanasbsi/AppSource/actions/runs/6905165127/job/18788521262#step:5:27)86-44[37](https://github.com/fofanasbsi/AppSource/actions/runs/6905165127/job/18788521262#step:5:38)-8937-055075aff4[43](https://github.com/fofanasbsi/AppSource/actions/runs/6905165127/job/18788521262#step:5:45).ps1: line 4 <- at <ScriptBlock>, <No file>: line 1
Error: Process completed with exit code 1.

**Comment by Steven-SBSI** [comment]
Run microsoft/AL-Go-Actions/Deliver@v4.0
Run $errorActionPreference = "Stop"; $ProgressPreference = "SilentlyContinue"; Set-StrictMode -Version 2.0
Downloading BcContainerHelper latest version from CDN
Import from C:\ProgramData\BcContainerHelper\6.0.0\BcContainerHelper\BcContainerHelper.ps1
BcContainerHelper version 6.0.0
BC.HelperFunctions emits usage statistics telemetry to Microsoft
Applying settings from D:\a\AppSource\AppSource\.github\AL-Go-Settings.json
Applying settings from D:\a\AppSource\AppSource\.AL-Go\settings.json
No settings found in D:\a\AppSource\AppSource\.github\Publish To AppSource.settings.json
No settings found in D:\a\AppSource\AppSource\.AL-Go\Publish To AppSource.settings.json
No settings found in D:\a\AppSource\AppSource\.AL-Go\fofanasbsi.settings.json
Artifacts latest
Projects:
.
Project 'AppSource'
Analyzing artifacts matching *-main-*-*
https://api.github.com/repos/fofanasbsi/AppSource/actions/artifacts?per_page=100&page=1
Matching artifacts:
- AppSource-main-Apps-1.0.19.0
Downloading artifact AppSource-main-Apps-1.0.19.0
https://api.github.com/repos/fofanasbsi/AppSource/actions/artifacts/1057241345/zip
D:\a\AppSource\AppSource\.artifacts\AppSource-main-Apps-1.0.19.0.zip
Analyzing artifacts matching *-main-*-*
https://api.github.com/repos/fofanasbsi/AppSource/actions/artifacts?per_page=100&page=1
Matching artifacts:
- No matching artifacts found
WARNING: Could not find any Dependencies artifacts for projects *, version latest
Artifacts:
- AppSource-main-Apps-1.0.19.0
Applying settings from D:\a\AppSource\AppSource\.github\AL-Go-Settings.json
Applying settings from D:\a\AppSource\AppSource\.AL-Go\settings.json
No settings found in D:\a\AppSource\AppSource\.github\Publish To AppSource.settings.json
No settings found in D:\a\AppSource\AppSource\.AL-Go\Publish To AppSource.settings.json
No settings found in D:\a\AppSource\AppSource\.AL-Go\fofanasbsi.settings.json
Analyzing repository
  Checking type
  Checking appFolders, testFolders and bcptTestFolders
  Application Dependency 2***.0
  Updating app- and test Dependencies
  Analyzing Test App Dependencies
No performance test apps found in bcptTestFolders in .AL-Go\settings.json
Azure.Storage Module is available in version 2.1.0
Using Azure.Storage version 2.1.0

Authentication failed
AppSource MainAppFolder .\AppSourceWorkFlow
Main App File:
- MohamedFOFANA_AppSourceWorkFlow_1.0.19.0.app
Library App Files:
- None
Submitting to AppSource
AL-Go action ran: Deliver Telemetry Correlation Id: 2a9c1a20-13e2-46ee-9381-f314d6c5e66d
Error: Unexpected error when running action. Error Message: Cannot bind argument to parameter 'authContext' because it is null., StackTrace: at <ScriptBlock>, D:\a\_actions\microsoft\AL-Go-Actions\v4.0\Deliver\Deliver.ps1: line 467 <- at <ScriptBlock>, D:\a\_temp\1b3f568e-2686-4[43](https://github.com/fofanasbsi/AppSource/actions/runs/6905165127/job/18788521262#step:5:45)7-8937-055075aff[44](https://github.com/fofanasbsi/AppSource/actions/runs/6905165127/job/18788521262#step:5:46)3.ps1: line 4 <- at <ScriptBlock>, <No file>: line 1
Error: Process completed with exit code 1.
